### PR TITLE
Add test and needed utilities for simple return creation

### PR DESCRIPTION
**Pull request by rue-kcarlton**

### CODE REVIEW

File: test/ui/simple-order.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          product_id: product.id,
+          quantity: 1,
+          fulfillable_quantity: 1,
+          fulfillment_status: 'fulfilled',
+          title: product.title,
+          variant_title: product.variant_title,
+          price: product.price,
+          sku: product.sku,
+          barcode: product.barcode,
+          vendor: '1999',
```
Comment (rue-kcarlton) [inline test/ui/simple-order.test.js:46]: Using a vendor that doesn't exist in Mirakl keeps them from potentially showing up in any reports that may incidentally run in non-prod AND keeps us incidents from being opened

File: test/ui/simple-order.test.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    // confirm it is enabled
+    expect(reasonSelectEnabled).toEqual(true);
+    // select first return reason
+    await selectUtil.selectByValue(reasonSelectElem, '1');
+    // click submit
+    const submitButton = await driver.findElement(By.css('button'));
+    await submitButton.click();
+
+    // longer explicit wait because creating the return takes time
+    const returnIdSpan = await driver.wait(until.elementLocated(By.css('.return-id')), 2000000);
```
Comment (rgg-amahajan) [inline test/ui/simple-order.test.js:97]: @rue-kcarlton is this indeed 2000 seconds (or did you plan this to be 200 seconds for the element to appear?) 😄 just a question

File: test/ui/simple-order.test.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    const returnId = await returnIdSpan.getAttribute('innerHTML');
+    expect(/^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i.test(returnId)).toEqual(true);
```
Comment (rgg-amahajan) [inline test/ui/simple-order.test.js:99]: @rue-kcarlton could uuid.validate() be used? available since uuid@8.3 so available in this project

**Review by rgg-amahajan — COMMENTED** [review]
Looks good overall, just a couple of comments to take a look at!

File: test/ui/simple-order.test.js (hunk 3 above)
Comment (rue-kcarlton) [reply test/ui/simple-order.test.js:99]: It absolutely could be... I'll modify because that is a much easier read

**Review by rgg-amahajan — APPROVED** [review]
Looks good!!

**Pull request merged**